packages/react-theme/src/global/fonts.ts

```diff
@@ -42,7 +42,8 @@ export const fontWeights: FontWeights = {
 
 // TODO find asset urls
 export const fontFamilies: FontFamilies = {
-  base: '',
-  monospace: '',
-  numeric: '',
+  base:
+    "'Segoe UI', 'Segoe UI Web (West European)', -apple-system, BlinkMacSystemFont, Roboto, 'Helvetica Neue', sans-serif",
+  monospace: "Consolas, 'Courier New', Courier, monospace",
+  numeric: 'Bahnschrift, $fontFamily-base',
```

Comment:
What is the `$fontFamily-base`? Should it be the `base` expanded here?

Reply:
Was updated in #16844.